### PR TITLE
Mainly fix and complete viselements.json

### PR DESCRIPTION
**Pull request by FabienLelaquais**

### CODE REVIEW

File: taipy/gui/viselements.json [hunk 1]
```diff
@@ -556,7 +556,7 @@
                         "name": "columns",
                         "type": "Union[str,list[str],dict[str,dict[str,str]]]",
                         "default_value": "<i>All columns</i>",
```
Comment (FredLL-Avaiga) [inline taipy/gui/viselements.json:558]: I thought you wanted to remove this ?

File: taipy/gui/viselements.json [hunk 2]
```diff
@@ -756,7 +756,7 @@
                         "name": "columns",
                         "type": "Union[str,list[str],dict[str,dict[str,Union[str,int]]]]",
                         "default_value": "<i>shows all columns when empty</i>",
```
Comment (FredLL-Avaiga) [inline taipy/gui/viselements.json:758]: or this

File: taipy/gui/viselements.json [hunk 3]
```diff
@@ -846,13 +846,14 @@
                     {
                         "name": "editable[<i>column_name</i>]",
                         "type": "bool",
-                        "default_value": "editable",
-                        "doc": "Indicates, if False, that the indicated column cannot be edited when editable is True."
+                        "default_value": "<i>editable</i>",
```
Comment (FredLL-Avaiga) [inline taipy/gui/viselements.json:849]: or is the i tag the way to remove it automagically ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: taipy/gui/viselements.json (hunk 3 above)
Comment (FabienLelaquais) [reply taipy/gui/viselements.json:849]: No. It's the convention for referring to a property name.

File: taipy/gui/viselements.json (hunk 2 above)
Comment (FabienLelaquais) [reply taipy/gui/viselements.json:758]: No we're good.

File: taipy/gui/viselements.json (hunk 1 above)
Comment (FabienLelaquais) [reply taipy/gui/viselements.json:558]: No we're good.

**Pull request merged**